kas/vendor/raspberrypi.yml

```diff
@@ -11,3 +11,4 @@ repos:
 local_conf_header:
   raspberrypi: |
     ROOTFS_IMAGE_EXTRA_INSTALL:append = " packagegroup-avocado-raspberrypi"
+    SDK_PKG_EXTRA_INSTALL:append = " packagegroup-avocado-raspberrypi-sdk"
```

Comment:
SDK_PKG_EXTRA_INSTALL gets populated here. The intention is that any target / platform specific tools that may need to be present can go in this.